src/components/typography/typography.component.tsx

```diff
@@ -39,6 +40,8 @@ export interface TypographyProps extends SpaceProps, TagProps {
   children?: React.ReactNode;
   /** The visual style to apply to the component */
   variant?: VariantTypes;
+  /** Override the ref value */
+  ref?: React.Ref<HTMLElement>;
```

Comment:
**issue**: I assume this was added so we could have the forward ref documented? 

Unfortunately this will actually add support for a new `ref` React prop that will be separate to the forwarded ref.

```suggestion
```

Due to how the typing of `React.forwardRef()` is defined, the type of the component it returns should automatically type the forwarded `ref`.

However, storybook won't be able to generate any documentation for the `ref`, so we may need to just document it via a story as a workaround.

Reply:
This work has been removed from the PR so it's no longer applicable